submarine-sdk/pysubmarine/submarine/store/sqlalchemy_store.py

```diff
@@ -0,0 +1,155 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from contextlib import contextmanager
+import math
+import sqlalchemy
+
+from submarine.exceptions import SubmarineException
+from submarine.utils import extract_db_type_from_uri
+from submarine.store.database.models import Base, SqlMetric, SqlParam
+from submarine.store.abstract_store import AbstractStore
+
+
+_logger = logging.getLogger(__name__)
+
+
+class SqlAlchemyStore(AbstractStore):
+    """
+    SQLAlchemy compliant backend store for tracking meta data for Submarine entities. Submarine
+    supports the database dialects ``mysql``, ``mssql``, ``sqlite``, and ``postgresql``.
```

Comment:
I know that SqlAlchemyStore can support many databases type, but the submarine project only supports MySQL database. We also do system testing for MySQL. In order to avoid confusion, we only write support MySQL.

I mean is change to `supports the database dialects ``mysql``.`